### PR TITLE
Moved MasterScore to separate file

### PR DESCRIPTION
**Pull request by igorkorsukov**

### CODE REVIEW

**Review by vpereverzev — APPROVED** [review]

File: src/engraving/libmscore/segment.cpp [hunk 1]
```diff
@@ -2378,6 +2382,10 @@ qreal Segment::elementsBottomOffsetFromSkyline(int staffIndex) const
     Ms::SkylineLine south = staffSystem->skyline().south();
     int bottomOffset = INT_MIN;
     for (Ms::SkylineSegment segment: south) {
+        Segment* seg = prev1enabled();
+        if (!seg) {
+            continue;
+        }
         bool ok = prev1enabled()->pagePos().x() <= segment.x && segment.x <= pagePos().x();
```
Comment (cbjeukendrup) [inline src/engraving/libmscore/segment.cpp:2389]: Shouldn't this be 
```cpp
bool ok = seg->pagePos().x() <= segment.x && segment.x <= pagePos().x();
```
?

Comment (igorkorsukov) [reply src/engraving/libmscore/segment.cpp]: Thanks, fixed

File: src/engraving/compat/readscorehook.cpp [hunk 2]
```diff
@@ -22,7 +22,8 @@
 #include "readscorehook.h"
 #include "readstyle.h"
 
-#include "libmscore/score.h"
+#include "libmscore/masterscore.h"
+#include "libmscore/masterscore.h"
```
Comment (cbjeukendrup) [inline src/engraving/compat/readscorehook.cpp:26]: Twice the same #include here??

File: src/engraving/compat/readstyle.cpp [hunk 3]
```diff
@@ -24,7 +24,8 @@
 #include "style/defaultstyle.h"
 #include "style/style.h"
 
-#include "libmscore/score.h"
+#include "libmscore/masterscore.h"
+#include "libmscore/masterscore.h"
```
Comment (cbjeukendrup) [inline src/engraving/compat/readstyle.cpp:28]: And here

File: src/engraving/compat/readscorehook.cpp (hunk 2 above)
Comment (igorkorsukov) [reply src/engraving/compat/readscorehook.cpp:26]: Oops )
I added `#include "libmscore/masterscore.h"` first
And after I decided to just change `#include "libmscore/score.h"` to `#include "libmscore/masterscore.h"`

Comment (igorkorsukov) [reply src/engraving/compat/readscorehook.cpp]: Fixed

File: src/engraving/compat/readstyle.cpp (hunk 3 above)
Comment (igorkorsukov) [reply src/engraving/compat/readstyle.cpp]: Fixed

File: src/engraving/libmscore/mcursor.cpp [hunk 4]
```diff
@@ -32,11 +32,13 @@
 #include "libmscore/durationtype.h"
 #include "libmscore/measure.h"
 #include "libmscore/segment.h"
-#include "libmscore/score.h"
+#include "libmscore/masterscore.h"
 #include "libmscore/instrtemplate.h"
 #include "libmscore/keysig.h"
 #include "libmscore/timesig.h"
 
+#include "masterscore.h"
```
Comment (cbjeukendrup) [inline src/engraving/libmscore/mcursor.cpp:40]: This one seems unnecessary too, I guess it's the same problem :)

File: src/importexport/imagesexport/internal/pdfwriter.cpp [hunk 5]
```diff
@@ -24,7 +24,8 @@
 
 #include <QPdfWriter>
 
-#include "libmscore/score.h"
+#include "libmscore/masterscore.h"
+#include "libmscore/masterscore.h"
```
Comment (cbjeukendrup) [inline src/importexport/imagesexport/internal/pdfwriter.cpp:28]: Another one...

File: src/engraving/libmscore/mcursor.cpp (hunk 4 above)
Comment (igorkorsukov) [reply src/engraving/libmscore/mcursor.cpp:40]: Needed, because

File: src/importexport/imagesexport/internal/pdfwriter.cpp (hunk 5 above)
Comment (igorkorsukov) [reply src/importexport/imagesexport/internal/pdfwriter.cpp]: Tanks, fixed

File: src/engraving/libmscore/mcursor.cpp (hunk 4 above)
Comment (cbjeukendrup) [reply src/engraving/libmscore/mcursor.cpp:40]: But there is already `#include "libmscore/masterscore.h"` at line 35.

Comment (igorkorsukov) [reply src/engraving/libmscore/mcursor.cpp:38]: Now, I got it, fixed )

**Pull request merged**